lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -44,6 +44,12 @@
     <% mode_class = '' %>
   % endif
 <div class="course-container">
+ % if course_program_info:
+  <div class="label-xseries-association">
+    <i class="xseries-icon"></i>
+    <p class="message-copy">${_("XSeries Program Course")}</p>
```

Comment:
actually @awais786 maybe a better idea would be to add `and course_program_info['category']=='xseries'` to the conditional, as you did on L351 below.  The idea is to be explicit about things that are getting hardcoded for XSeries.